lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -44,6 +44,16 @@ export class BrowserUtils {
         return window.parent !== window;
     }
 
+    /**
+     * Returns boolean of whether or not the current window is a popup opened by msal
+     */
+    static isInPopup(): boolean {
+        return !!window.opener && 
```

Comment:
Might be good to add `typeof window !== "undefined"` so that it doesn't break if called server side.

Reply:
Done